README.md

```diff
@@ -63,6 +63,12 @@ A good starting point for troubleshooting is:
 
 If you're still having trouble, post an issue so we can look into it.
 
+## Tested with the following RN versions
+
+Folks using this module have confirmed the library works with the following React Native versions. If your version is not on the list and you have confirmed it works as expected please create a PR (you can even do it on github.com)
+
+- 0.42.0
```

Comment:
If they just moved the file around, then it won't matter, as we bundle our own version. I'll keep an eye on this an ensure it continues to work post 0.46.